packages/terra-functional-testing/src/services/wdio-visual-regression-service/modules/getTerraFormFactor.js

```diff
@@ -1,5 +1,26 @@
+/* global browser */
+
 const { TERRA_VIEWPORTS } = require('../../../constants');
-const { getViewportSize } = require('../../../commands/utils');
+
+/**
+ * Determines the current viewport size.
+ *
+ * @returns {Object} - the current viewport size.
+ */
+function getViewportSize() {
+  // eslint-disable-next-line prefer-arrow-callback
+  const resolution = browser.execute(function getResolution() {
+    return {
+      screenWidth: Math.max(document.documentElement.clientWidth, window.innerWidth || 0),
+      screenHeight: Math.max(document.documentElement.clientHeight, window.innerHeight || 0),
+    };
+  });
+
+  return {
+    width: resolution.screenWidth,
+    height: resolution.screenHeight,
+  };
+}
```

Comment:
Could we just do this?
```suggestion
function getViewportSize() {
  // eslint-disable-next-line prefer-arrow-callback
  return global.browser.getWindowSize();
}
```

Reply:
I think we could and probably should since we use [setWindowSize](https://github.com/cerner/terra-toolkit/pull/674/files#diff-cf781d47655d684c44cf4d2d28591066268e50b8f239a7250e0b0228ded7c9ffR18) to set the window size so calling `getWindowSize` would be more consistent and less code/complex. I also think changing to `getWindowSize` has no impact on what "formFactor" is returned from `getTerraFormFactor` below where this is being used.

If we wanted to we could even get rid of this function and just call `global.browser.getWindowSize().width` directly in [here](https://github.com/cerner/terra-toolkit/pull/674/files#diff-a204538639999723fcde3efa5da7ab19229d476d2d255839f5f9671610cafdadR32) but the proposed change is just fine.